nuxeo-java-client-test/src/test/java/org/nuxeo/client/test/TestUpload.java

```diff
@@ -118,6 +125,13 @@ public void itCanUploadChunks() {
         assertEquals(file.length(), batchFile.getSize());
         assertEquals(4, batchFile.getChunkCount());
         assertEquals(batchFile.getChunkCount(), batchFile.getUploadedChunkIds().length);
+        // Check the uploaded blob
+        org.nuxeo.ecm.core.api.Blob uploadedBlob = Framework.getService(BatchManager.class).getBlob(
+                batchUpload.getBatchId(), batchUpload.getFileIdx());
+        assertNotNull(uploadedBlob);
+        try (InputStream is = new FileInputStream(file)) {
+            assertEquals(DigestUtils.md5Hex(is), DigestUtils.md5Hex(uploadedBlob.getStream()));
```

Comment:
`uploadedBlob.getStream()` should be closed as well, so you can put it with the other stream in the `try`